src/Models/BaseElement.php

```diff
@@ -694,14 +699,48 @@ public function getSummary()
         return '';
     }
 
+
+    /**
+     * @return array
+     */
+    public function getSummaryData()
+    {
+        $summaryData = [];
+
+        $this->extend('setDefaultSummaryData', $summaryData);
+
+        return $summaryData;
+    }
+
+    /**
+     * This can be overridden on child elements.
+     *
+     * @return string
+     */
+    public function getContent()
+    {
+        return '';
+    }
+
+    /**
+     * This can be overridden on child elements.
+     *
+     * @return string
+     */
+    public function getFileURL()
+    {
+        return '';
+    }
```

Comment:
I think summary data should mean we can remove this now. Doing so might mean that the file block needs a custom extension for the `Element` component, but that would also be a good thing for us dogfooding this architecture